hat/app/org/hatdex/hat/api/controllers/RichData.scala

```diff
@@ -83,8 +83,11 @@ class RichData @Inject() (
     configuration.underlying.getString("adjudicator.address")
   val adjudicatorScheme =
     configuration.underlying.getString("adjudicator.scheme")
-  val adjudicatorEndpoint = s"${adjudicatorScheme}${adjudicatorAddress}"
-  val adjudicatorClient = new AdjudicatorRequest(adjudicatorEndpoint, wsClient)
+  val adjudicatorEndpoint =
```

Comment:
what is the appropriate access for these ? do you really want them public ? If so public members should have an explicit type https://dswift.atlassian.net/wiki/spaces/DSENGINEER/pages/4751401/Best+Practices#Use-explicit-types